web/src/stores.ts

```diff
@@ -97,3 +99,13 @@ export let useLocalVite: Writable<boolean> = writable(false);
 export function assetUrl(path: string): string {
   return get(useLocalVite) ? `/${path}` : `https://assets.od2net.org/${path}`;
 }
+
+export function returnToChooseProject() {
+  mode.set({ mode: "title", firstLoad: false });
+
+  let bounds = [-180, -90, 180, 90] as LngLatBoundsLike;
+  if (get(appFocus) == "cnt") {
+    bounds = [-8.943, 54.631, -0.901, 59.489];
+  }
+  get(map)?.fitBounds(bounds, { duration: 500 });
```

Comment:
500 is maybe a little fast, but anything too long gets annoying. What do you think?

Reply:
It feels maybe a little fast (definitely not too slow!), but not unreasonable. Let's go with it and we can always tweak it later.